### PR TITLE
[master<] MAGE changelog 1.8.0

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Changelog for MAGE v.1.8.0

### CODE REVIEW

File: mage/changelog.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.8.0 - Jun 28, 2023
```
Comment (vpavicic) [inline mage/changelog.md:7]:
```suggestion
## v1.8.0 - Jun 30, 2023
```

File: mage/changelog.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ title: Changelog
...
+  suited for large language models (LLMs).
+  [#225](https://github.com/memgraph/mage/pull/225)
+- When executing complex queries, the `period` module allows batching results
+  from one query into another to improve execution time.
+  [#221](https://github.com/memgraph/mage/pull/221)
+- The `conditional_execution` module which allows the execution of different
+  queries depending on certain conditions being met, has been rewritten from
+  Python to C++ to improve performance and it can also periodically iterates.
+  [#222](https://github.com/memgraph/mage/pull/222)
+- The `migration` module has an option to get data from MySQL, SQL server, or
```
Comment (antoniofilipovic) [inline mage/changelog.md:21]: `migrate` module

**Review by antoniofilipovic — CHANGES_REQUESTED** [review]
only migrate module is wrongly called

File: mage/changelog.md (hunk 2 above)
Comment (vpavicic) [inline mage/changelog.md:21]:
```suggestion
- The `migrate` module has the option to get data from MySQL, SQL server, or
```

**Pull request merged**